src/main/resources/database-init.sql

```diff
@@ -0,0 +1,49 @@
+DROP TABLE IF EXISTS orders;
+DROP TABLE IF EXISTS users;
+DROP TABLE IF EXISTS stations;
+DROP TABLE IF EXISTS carriers;
+
+CREATE TABLE users
+(
+    id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    username VARCHAR(50) NOT NULL UNIQUE,
+    first_name VARCHAR(255),
+    last_name VARCHAR(255),
+    password VARCHAR(100) NOT NULL,
+    phone VARCHAR(15)
+);
+
+CREATE TABLE stations
+(
+    id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    city VARCHAR(50) NOT NULL,
+    address VARCHAR(255) NOT NULL,
+    status ENUM('unavailable', 'available') NOT NULL,
+    latitude VARCHAR(10) NOT NULL,
```

Comment:
I guess lat and lng should be FLOAT here

Reply:
As a reference of decimal degrees, check following:
https://developers.google.com/maps/documentation/javascript/examples/event-click-latlng#maps_event_click_latlng-javascript